index.html

```diff
@@ -20,43 +20,222 @@
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </article>
+        <article class="other-projects">
+          <h1 class="title-post">Profesional Art Printing Data</h1>
+          <h2 class="supporting-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </h2>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </article>
+        <article class="other-projects">
+          <h1 class="title-post">Profesional Art Printing Data</h1>
+          <h2 class="supporting-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </h2>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </article>
+        <article class="other-projects">
+          <h1 class="title-post">Profesional Art Printing Data</h1>
+          <h2 class="supporting-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </h2>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </article>
+        <article class="other-projects">
+          <h1 class="title-post">Profesional Art Printing Data</h1>
+          <h2 class="supporting-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </h2>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </article>
+      </section>
+
+      <section class="about-me">
+        <h1 class="primary-text">About me</h1>
+        <p class="description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button>Get My Resume</button>
+        <hr />
+        <h2>🔶 Language</h2>
+        <h3>Javascript</h3>
+        <h3>Ruby</h3>
+        <h3>Html</h3>
+        <h3>Css</h3>
+        <h2>🟧 Frameworks</h2>
+        <h3>Bootstrap</h3>
+        <h3>Ruby on Rails</h3>
+        <h3>RSPec</h3>
+        <h3>CapyBara</h3>
+        <h3>Selenium</h3>
+        <h2>🟠 Skills</h2>
+        <h3>Codekit</h3>
+        <h3>Github</h3>
+        <h3>Codepen</h3>
+        <h3>Gitlab</h3>
+        <h3>Terminal</h3>
```

Comment:
- I think it is important that The technology categories ("Languages", "Frameworks" and "Skills") are in a `<ul>`, each of them in a `<li>`

- Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `<ul>` and each of them a `<li>`

You can simply re-factor your code to look something like this 👇🏽 
```
  <ul class="languages">
          <li class="frame">
            <h2>🔶Languages</h2>
          </li>
          <ul class="list">
            <li>javascript</li>
            <li>ruby</li>
            <li>html</li>
            <li>css</li>
          </ul>
     </ul>
    <ul class="Frameworks">
          <li>
            <h2>🟧 Frameworks</h2>
          </li>
          <ul class="list">
            <li>Bootstrap</li>
            <li>Ruby on Rails</li>
            <li>RSPec</li>
            <li>CapyBara</li>
            <li>Selenium</li>
          </ul>
      </ul>
```

- What do you think about this code snippet that I have added above?  You should include the rest of the list and can modify the class name as you wish but kindly remember to update your CSS file for the list tag as well. 🍒